src/components/Details.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 
 export default class Details extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {message: 'Hello, this will be the details page for each Movie & TV show'};
```

Comment:
I'd suggest you to always add an empty space separating curly brackets, for readability purposes